### PR TITLE
Django 1.9+ support.

### PR DESCRIPTION
**Pull request by meric**
I've made some changes I needed to get this module "running" for the current `master` branch django, while maintaining backwards compatibility with older django versions.

Thought I might submit a pull request, though I don't know if it's wise to do this before the next version of django is tagged.

Also, #23.

### CODE REVIEW

File: django_comments/__init__.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 from django.core.exceptions import ImproperlyConfigured
-from django.utils.importlib import import_module
 
-from django_comments.models import Comment
-from django_comments.forms import CommentForm
+try:
+    from django.utils.importlib import import_module
+except ImportError:
+    # Django 1.9+
+    from importlib import import_module
```
Comment (collinanderson) [inline django_comments/__init__.py]: I'd reverse this. try plain importlib first.

File: django_comments/__init__.py [hunk 2]
```diff
@@ -35,6 +38,7 @@ def get_comment_app_name():
     return getattr(settings, 'COMMENTS_APP', DEFAULT_COMMENTS_APP)
 
 def get_model():
+    from django_comments.models import Comment
```
Comment (collinanderson) [inline django_comments/__init__.py]: I had to do the same thing. I think this is the right approach.

File: django_comments/forms.py [hunk 3]
```diff
@@ -1,6 +1,11 @@
 import time
 from django import forms
-from django.forms.util import ErrorDict
+
+try:
+    from django.forms.util import ErrorDict    
+except ImportError:
+    from django.forms.utils import ErrorDict
```
Comment (collinanderson) [inline django_comments/forms.py]: Again, try utils first, then fallback to util

File: django_comments/models.py [hunk 4]
```diff
@@ -1,5 +1,10 @@
 from django.conf import settings
-from django.contrib.contenttypes import generic
+
+try:
+    from django.contrib.contenttypes.generic import GenericForeignKey
+except ImportError:
+    from django.contrib.contenttypes.fields import GenericForeignKey
```
Comment (collinanderson) [inline django_comments/models.py]: try fields first and fallback to generic.

**Comment by collinanderson** [comment]
Otherwise looks good to me at least. :) Thanks for the patch!

**Comment by meric** [comment]
@collinanderson Thanks for looking through it! I've made the respective changes. How is it now? Let me know if you want me to rebase/squash the commits!

**Comment by collinanderson** [comment]
This looks good to me.

**Comment by claudep** [comment]
Thanks, pushed in ad2a4ed8dfe61176b4233484c52428fa93618e76

**Pull request closed without merging**